pkg/transport/transport_test.go

```diff
@@ -74,6 +74,14 @@ func TestDialNode(t *testing.T) {
 				},
 				Type: pb.NodeType_STORAGE,
 			},
+			{
+				Id: storj.NodeID{},
+				Address: &pb.NodeAddress{
+					Transport: pb.NodeTransport_TCP_TLS_GRPC,
+					Address:   "127.0.0.1:100",
```

Comment:
By "valid address", I meant an address that can actually respond, e.g. `planet.StorageNodes[1].Addr()`.

Reply:
My bad... actually done now.